### DISCUSSION TITLE
Two factor auth with harware security keys don't work in Firefox

### DISCUSSION THREAD

**Discussion by konclave** [community/community#31879] [Other Feature Feedback, Questions, & Ideas] [answered]
After clicking `Use security key` in 2FA prompt, I'm getting an error message: `Security key authentication failed.`. 
In debugger console I see `Uncaught (in promise) DOMException: The operation failed for an unknown transient reason` error thrown in `webauth-get.ts:110:19`. 
It fails in `prompt` function executing `const signResponse = await get(parseRequestOptionsFromJSON(json))` expression.

**Comment by jfullstackdev** [comment, marked as answer]
do you have Chrome ? maybe you can try that

**Reply by konclave** [reply]
No, I don't have Chrome, but it works in Brave, which is based on Chromium.
But I'd like to make it work in FF because it's my main browser.

**Reply by xchapron-ledger** [reply]
WebAuthn support is under development for Firefox on MAC and Linux: https://bugzilla.mozilla.org/show_bug.cgi?id=1530370
